python/in3/client.py

```diff
@@ -69,7 +69,7 @@ def ens_namehash(self, domain_name: str) -> str:
             node (str): Formatted string referred as `node` in ENS documentation
         """
         if not isinstance(domain_name, str) or not re.match(r'(\w+.eth$)', domain_name):
-            raise AssertionError('Client: ENS domain name must end with .ens')
+            raise AssertionError('Client: ENS domain name must end with .eth')
```

Comment:
I think should be re-used as it's own exception. However, still getting used to the format of this project.

Reply:
thanks, good catch